src/main/java/com/spider/solomon/reporting/ReportController.java

```diff
@@ -0,0 +1,26 @@
+package com.spider.solomon.reporting;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import jakarta.annotation.Resource;
+
+/**
+ * Endpoints for general reporting
+ */
+@RestController
+@RequestMapping("/reports")
+public class ReportController {
+	@Resource
+	TrendAnalyzer trendAnalyzer;
+
+	/**
+	 * Returns the trend of the total sales data
+	 */
+	@RequestMapping(value = "/trend", method = RequestMethod.GET)
+	public String trend () {
```

Comment:
[Meta: I might suggest removing the space between `trend` and `()`. Ideally i'd never comment on such a thing because it'd be enforced with an auto-formatter. Also I might not comment because looking at the code, it seems like some methods do use a space like this, while others do not, so the convention is unclear. This is a super nitpick thing so I'd never block a merge for it.]